src/Mod/Sketcher/Gui/DrawSketchHandlerArcOfHyperbola.h

```diff
@@ -224,7 +226,9 @@ class DrawSketchHandlerArcOfHyperbola : public DrawSketchHandler
 
             if (boost::math::isnan(startAngle) || boost::math::isnan(endAngle)) {
                 sketchgui->purgeHandler();
-                Base::Console().Error("Cannot create arc of hyperbola from invalid angles, try again!\n");
+                Gui::NotifyError(sketchgui,
+                                    QT_TRANSLATE_NOOP("Notifications", "Error"),
+                                    QT_TRANSLATE_NOOP("Notifications", "Cannot create arc of hyperbola from invalid angles, try again!"));

```

Comment:
"Try again!" might be out of place here :). Then again, how often are these angles actually NaN? I gather this is a rare error...

Reply:
I thought that out a couple of times.

This kind of expressions are generally out of place. It may even appear as misleading: "If I did it once and got an error, if I do again I would get an error to, wouldn't I?". The problem is that... most probably not, because not the same exact points are going to be hit. So perhaps trying again is not a bad piece of advice after all.

That is why I left it that way. But as you say, I have never seen this error popping...